contrib/cloud.google.com/go/pubsub.v1/example_test.go

```diff
@@ -0,0 +1,39 @@
+package pubsub
```

Comment:
Please take a look at the other `example_test.go` files. We need this to be a different package so that whoever views the code would see the example as they would write it.
```suggestion
package pubsub_test
```